### PR TITLE
Broader dependencies as recommended in #12

### PR DESCRIPTION
**Pull request by timgates42**
Following up on the good suggestion in that PR.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -30,7 +30,7 @@ Repository = "https://github.com/NiklasRosenstein/python-github-bot-api"
 [tool.poetry.dependencies]
 python = "^3.8"
 "nr.functional" = "^0.2.0"
-cryptography = "^39.0.2"
+cryptography = "*"
```
Comment (NiklasRosenstein) [inline pyproject.toml]:
```suggestion
cryptography = ">=39.0.2"
```

Should we do this to lower bound at least on a known old working version of the package?

Comment (eugenecg) [reply pyproject.toml]: The latest version without any vulnerabilities is 42.0.5 shall we use the below perhaps? 

`"^42.0.5"`

Comment (eugenecg) [inline pyproject.toml]:
```suggestion
cryptography = "^42.0.5"
```

**Comment by NiklasRosenstein** [comment]
Released in `0.6.0` (as minor because of removal of the deprecated `GithubApp.client`)

**Pull request merged**